api/auth/facebookConfig.js

```diff
@@ -0,0 +1,10 @@
+module.exports={
+    "facebook_api_key"      :     "FB APP ID",
+    "facebook_api_secret"   :     "FB API SECRET",
+    "callback_url"          :     "http://localhost:3000/auth/facebook/callback",
+    "use_database"          :      false,
+    "host"                  :     "localhost",
+    "username"              :     "root",
+    "password"              :     "",
+    "database"              :     "Database Name"
+  }
```

Comment:
This file isn't being used. We can probably delete it.
https://github.com/labs14-market-organizer/backend/blob/68a57d39cfb574049223d49d3b3c130c4384189b/api/auth/facebookConfig.js#L1-L10